app/assets/stylesheets/master.css.scss

```diff
@@ -0,0 +1,21 @@
+.hero-image {
+  background: url('/assets/background-img.jpeg') no-repeat center center fixed;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background, background-attachment, -moz-background-size, -o-background-size, -webkit-background-size, background-size<br>Avoid vendor prefixes.